Docs: add "GTO with MLEM" page (#259)
iterative/mlem.ai: content/docs/gto/command-reference/describe.md

}
```

The output is in JSON format for ease of parsing programmatically.

> Note, that for local repos the `artifacts.yaml` is read from the workspace
> without Git, so if you have uncommitted changes, they will be reflected in the
> output. If you want to read from specific commit, you need to specify `--rev`
> option.
> 
> You can also get annotation for specific versions (these are the same shortcuts

Reviewer: The first part is expected, I think. We can simplify this

```suggestion
If you want to read information from previous commits, specify a `--rev`.
You can also get annotation for specific artifact versions (same shortcuts
```
